src/Chainweb/Version.hs

```diff
@@ -208,6 +209,15 @@ instance IsMerkleLogEntry ChainwebHashTag ChainwebVersion where
     {-# INLINE toMerkleNode #-}
     {-# INLINE fromMerkleNode #-}
 
+-- | Display the `ChainwebVersion` without any Chain Graph information.
```

Comment:
Why is this needed? It may cause confusion when two different versions are displayed equal.

Reply:
I've removed it.